docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -2,75 +2,93 @@
 
 This guide contains general tips on how to investigate an application deployment that doesn't work correctly.
 
-## Clean up Terraform state
+## Cleanup Terraform state
+Terraform stores the lock and current state in `terraform-backend.tf`. This file is generated by the installation process based on the content of `config.tfvars`. During the installation, both the installer and Terraform generate some temporary files that are removed after the environment has been uninstalled.
 
-Terraform stores the lock and current state in `terraform-backend.tf`. 
-This file is generated by the installation process based on the content of `config.tfvars`.
-During the installation, both the installer and Terraform generate some temporary files that are removed after the environment has been uninstalled.
-
-If you installed an environment and then changed any mandatory configuration settings such as `region` or `environment_name`, then clean up 
-all temporary files generated by Terraform before installing a new environment. 
+If you installed an environment and then changed any mandatory configuration settings such as `region` or `environment_name`, then clean up all temporary files generated by Terraform before installing a new environment.
 
 To clean up local Terraform files, run the following command:
-
-```shell
+```
 ./pkg/scripts/cleanup.sh -t
 ```
 
 To clean up locally generated variable files, run the following command:
-
-```shell
+```
 ./pkg/scripts/cleanup.sh -s
 ```
-
 !!! tip
-    You can clean up both the local Terraform files and locally generated variable files by using the `-t` and `-s` switches in one command:
-    
-    ```shell
+    You can clean up both the local Terraform files and locally generated variable files by using the -t and -s switches in one command:
+    ```
     ./pkg/scripts/cleanup.sh -s -t
     ```
 
-!!! info
+!!! info 
     Temporary variable files will be regenerated using the configurating file and overriden during the installation process.
+    
 
-## Interrupted uninstallation
+## Uninstall - Invalid configuration file   
+  
+**Symptom**
+If you try to uninstall an environment by using a different configuration file than the one you used to install it or by using a different version of the code, you may encounter some issues during uninstallation. In most cases, Terraform reports that the resource cannot be removed because it's in use.
 
-### Symptom
+**Solution**
+Idenity the resource and delete it manually from the AWS console, and then restart the uninstallation process.
```

Comment:
Just wondering if it would be nicer to use an expand macro here? Something like...

```

???tip "Uninstall - Invalid configuration file "

**Symptom**
If you try to uninstall an environment by using a different configuration file than the one you used to install it or by using a different version of the code, you may encounter some issues during uninstallation. In most cases, Terraform reports that the resource cannot be removed because it's in use.

**Solution**
Idenity the resource and delete it manually from the AWS console, and then restart the uninstallation process.


```

Reply:
That is a very nice suggestion indeed. The reason I have subjects as a title is to include them in table of content. In this case when the content of this page is long (which is) then they can easily jump into the issue by clicking on TOC in right side of the page.